e2e-tests/runner/examples/connectors/PhantomSolanaConnector/setup.ts

```diff
@@ -0,0 +1,60 @@
+// Use a test fixture to set the context so tests have access to the wallet extension.
+import { downloadFuel } from '@fuels/playwright-utils';
+import type { BrowserContext } from '@playwright/test';
+import { test as base, chromium } from '@playwright/test';
+import phantomCommands from '../../../node_modules/@phantom/synpress/commands/phantom';
+import phantomHelpers from '../../../node_modules/@phantom/synpress/helpers';
+
+import { ETH_MNEMONIC, ETH_WALLET_PASSWORD } from './mocks';
+import { getExtensionsData } from './utils/getExtensionsData';
+import { waitForExtensions } from './utils/waitForExtensions';
+
+export const test = base.extend<{
+  context: BrowserContext;
+  extensionId: string;
+}>({
+  context: async ({ context: _ }, use) => {
+    // required for synpress
+    global.expect = expect;
+    // download fuel wallet
+    // download metamask
+    const phantomPath = await phantomHelpers.prepareProvider(
+      'phantom',
+      'latest',
+    );
+    // prepare browser args
+    const browserArgs = [
+      `--disable-extensions-except=${phantomPath}`,
+      `--load-extension=${phantomPath}`,
+      '--remote-debugging-port=9222',
+    ];
+    // launch browser
```

Comment:
I would remove those kind of comments that are already clear by reading the code itself